Modules/Logger/include/selxLogger.h

```diff
@@ -30,14 +30,15 @@
 namespace selx
 {
 
-enum class SeverityType {
-  SELX_TRACE,
-  SELX_DEBUG,
-  SELX_INFO,
-  SELX_WARNING,
-  SELX_ERROR,
-  SELX_FATAL
-};
+typedef enum {
```

Comment:
Remove typedef and replace with name. It will become `enum class LogLevel { ... }`. Class is preferable, remove if incompatible.